Sources/ArcGISToolkit/Components/FloorFilter/FloorFilterViewModel.swift

```diff
@@ -18,16 +18,6 @@ import SwiftUI
 /// Manages the state for a `FloorFilter`.
 @MainActor
 final class FloorFilterViewModel: ObservableObject {
-    ///  A selected site, floor, or level.
-    enum Selection: Hashable {
-        /// A selected site.
-        case site(FloorSite)
-        /// A selected facility.
-        case facility(FloorFacility)
-        /// A selected level.
-        case level(FloorLevel)
-    }
```

Comment:
Because this type is no longer used by only the model and is now public, it made sense to un-nest it and give it a new name, `FloorFilterSelection`, similar to `FloorFilterAutomaticSelectionMode`.